src/ReachSets/ContinuousPost/BFFPS19/reach_blocks.jl

```diff
@@ -0,0 +1,302 @@
+
+
+# sparse
+function reach_blocks!(ϕ::SparseMatrixCSC{NUM, Int},
+                       Xhat0::Vector{<:LazySet{NUM}},
+                       U::Union{ConstantInput, Nothing},
+                       overapproximate::Function,
+                       overapproximate_inputs::Function,
+                       n::Int,
+                       N::Union{Int, Nothing},
+                       output_function::Union{Function, Nothing},
+                       blocks::AbstractVector{Int},
+                       partition::AbstractVector{<:Union{AbstractVector{Int}, Int}},
+                       δ::NUM,
+                       guards_constraints_proj::Vector{HPolyhedron{NUM}},
```

Comment:
```suggestion
                       guards_constraints_proj::Vector{<:LazySet{NUM}},
```
The restriction to `HPolyhedron` is not needed.
Also better rename to `guards_proj` because the `constraints` is irritating.